int8 arg_one
int8 arg_two